.github/workflows/roman_ci_cron.yaml

```diff
@@ -0,0 +1,67 @@
+
+name: Weekly cron
+
+on:
+  schedule:
+    # Weekly Monday 6AM build
+    - cron: "0 0 * * 1"
+  workflow_dispatch:
+
+jobs:
+  crds:
+    name: retrieve current CRDS context
+    runs-on: ubuntu-latest
+    env:
+      OBSERVATORY: roman
+      CRDS_PATH: /tmp/crds_cache
+      CRDS_SERVER_URL: https://roman-crds.stsci.edu
+    steps:
+      - id: context
+        run: >
+          echo "pmap=$(
+          curl -s -X POST -d '{"jsonrpc": "1.0", "method": "get_default_context", "params": ["${{ env.OBSERVATORY }}"], "id": 1}' ${{ env.CRDS_SERVER_URL }}/json/ |
+          python -c "import sys, json; print(json.load(sys.stdin)['result'])"
+          )" >> $GITHUB_OUTPUT
+        # Get default CRDS_CONTEXT without installing crds client
+        # See https://hst-crds.stsci.edu/static/users_guide/web_services.html#generic-request
+      - id: path
+        run: echo "path=${{ env.CRDS_PATH }}" >> $GITHUB_OUTPUT
+      - id: server
+        run: echo "url=${{ env.CRDS_SERVER_URL }}" >> $GITHUB_OUTPUT
+    outputs:
+      context: ${{ steps.context.outputs.pmap }}
+      path: ${{ steps.path.outputs.path }}
+      server: ${{ steps.server.outputs.url }}
+  test:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@main
+    needs: [ crds ]
+    with:
+      setenv: |
+        CRDS_PATH: ${{ needs.crds.outputs.path }}
+        CRDS_SERVER_URL: ${{ needs.crds.outputs.server }}
+        CRDS_CLIENT_RETRY_COUNT: 3
+        CRDS_CLIENT_RETRY_DELAY_SECONDS: 20
+      cache-path: ${{ needs.crds.outputs.path }}
+      cache-key: crds-${{ needs.crds.outputs.context }}
+      envs: |
+        - macos: test-xdist
+          python-version: 3.9
+        - macos: test-sdpdeps-xdist
+          python-version: 3.9
+        - macos: test-xdist
+          python-version: 3.10
+        - linux: test-pyargs-xdist
+        - linux: test-devdeps-xdist
+          python-version: 3.8
```

Comment:
It looks like there was a commit to drop 3.8. https://github.com/spacetelescope/romancal/pull/612/commits/355374697f6759515c657b75804a03dfb1bac099
Should these jobs also be dropped and the minimum version bumped?

Reply:
Right now python 3.8 is the min python: https://github.com/spacetelescope/romancal/blob/f52caa3528c5f7dbf88a80414c9c750c55915f25/pyproject.toml#L5. I don't think a python version change should be part of a refactor of CI. That's out of scope.

Reply:
good point, I removed 3.8 tests because of an error but they seem to work now. Added back